#!/usr/bin/env python

PACKAGE = 'coral_usb'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('score_thresh', double_t, 0, 'Threshold for confidence score', 0.20, 0.0, 1.0)
gen.add('joint_score_thresh', double_t, 0, 'Threshold for confidence score for each joint', 0.20, 0.0, 1.0)

exit(gen.generate(PACKAGE, PACKAGE, 'EdgeTPUHumanPoseEstimator'))
